scripts/update_api_description.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+readonly THIS_DIR="$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd | xargs realpath)"
+readonly ROOT_DIR="${THIS_DIR}/.."
+
+function main() {
+	if [[ $# -ne 0 ]] ; then
+        echo "USAGE: $0"
+        exit 1
+    fi
+
+    trap exit SIGINT
+
+	cd "${ROOT_DIR}"
+
+	go run cmd/describe-api/main.go > api.json
```

Comment:
I don't think I like this file just hanging out in the top-level directory.
Maybe we should make a directory like "resources" and put it in there.

If we do that, we should probably move the version file as well.
Be careful, this requires changes to docker material as well (you should search for the filename).